clang-tools-extra/test/clang-tidy/checkers/modernize/loop-convert-extra.cpp

```diff
@@ -1062,7 +1062,7 @@ void f() {
     E Ee{ { { g( { Array[I] } ) } } };
   }
   // CHECK-MESSAGES: :[[@LINE-7]]:3: warning: use range-based for loop instead
-  // CHECK-FIXES: for (int I : Array)
+  // CHECK-FIXES: for (int & I : Array)
```

Comment:
My changes in SemaInit.cpp likely caused this, however the outcome seems correct to me, so I just changed the test.